andela/kingsmen-ah-frontend
# #166272289 Chore Set up redux

#### What does this PR do?
This PR sets up Redux into the application
#### Description of Task to be completed?
- Set up Redux to the application
#### How should this be manually tested?
- clone the repo
- install the packages via `npm install`
- start the dev-server with `npm run start:dev`
- install `Redux chrome devtool` as an extension on chrome
- Open your console and check for the `Redux` tab
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[#166272289](https://www.pivotaltracker.com/story/show/166272289)
#### Screenshots (if appropriate)
N/A
#### Questions:
N/A
